### PR TITLE
fix(website): add isPublished flag in blog page & add check on post page

### PR DESCRIPTION
**Pull request by hadzhehsen**

### CODE REVIEW

File: apps/website/pages/blog/index.tsx [hunk 1]
```diff
@@ -38,9 +43,11 @@ function mapStorybookToPosts(data: { posts: StorybookPost[] }): Post[] {
         content: post.content ?? '',
         image,
         isFeatured: post.featured,
+        isPublished: post.isPublished,
```
Comment (olegshilov) [inline apps/website/pages/blog/index.tsx]: remove, no need this here. we already filtered unpublished posts before

File: apps/website/pages/blog/index.tsx [hunk 2]
```diff
@@ -13,13 +13,18 @@ interface StorybookPost {
     filename: null | string;
   };
   featured: boolean;
+  isPublished: boolean;
```
Comment (olegshilov) [inline apps/website/pages/blog/index.tsx:16]: remove

**Review by olegshilov — CHANGES_REQUESTED** [review]

File: apps/website/pages/blog/index.tsx (hunk 1 above)
Comment (hadzhehsen) [reply apps/website/pages/blog/index.tsx]: Requested changes have been made

File: apps/website/pages/blog/index.tsx (hunk 2 above)
Comment (hadzhehsen) [reply apps/website/pages/blog/index.tsx:16]: Requested changes have been made

**Review by olegshilov — DISMISSED** [review]

**Review by kioqq — APPROVED** [review]

**Pull request merged**